app/controllers/Round.scala

```diff
@@ -212,41 +212,40 @@ final class Round(
   private[controllers] def getPlayerChat(game: GameModel, tour: Option[Tour])(using
       ctx: Context
   ): Fu[Option[Chat.GameOrEvent]] =
-    ctx.kid.no.so:
```

Comment:
this causes the game chat to always be loaded and sent to all clients, including the mobile apps, where kids will get to see the other player messages.

Reply:
I feared my approach would be too simple.
Let me think of something else.